ruby-challenges.rb

```diff
@@ -10,7 +10,16 @@
 num3 = 221
 # Expected output: '221 is odd'
 
-
+def odd_or_even (number)
+    if number % 2 == 0
+        'even'
+    else
+        'odd'
+    end
+end
```

Comment:
Nice!  Ruby also has a built in method that evaluates the statement on line 14 for you.